<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Hospitalière</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">

    <!-- QLabel pour afficher le logo -->
    <item>
     <widget class="QLabel" name="labelLogo">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>

    <!-- Bouton Gérer Patients -->
    <item>
     <widget class="QPushButton" name="btnGererPatients">
      <property name="text">
       <string>Gérer les Patients</string>
      </property>
     </widget>
    </item>

    <!-- Bouton Gérer Médecins -->
    <item>
     <widget class="QPushButton" name="btnGererMedecins">
      <property name="text">
       <string>Gérer les Médecins</string>
      </property>
     </widget>
    </item>

    <!-- Bouton Gérer Rendez-vous -->
    <item>
     <widget class="QPushButton" name="btnGererRendezVous">
      <property name="text">
       <string>Gérer les Rendez-vous</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
